packages/@sanity/cli/src/util/journeyConfig.ts

```diff
@@ -0,0 +1,270 @@
+import fs from 'fs/promises'
+import path from 'path'
+import {format} from 'prettier'
+import {Worker} from 'worker_threads'
+
+import {
+  type ArrayDefinition,
+  type BaseSchemaDefinition,
+  type DocumentDefinition,
+  type ObjectDefinition,
+} from '../../../types'
+import {type CliApiClient} from '../types'
+import {getCliWorkerPath} from './cliWorker'
+
+/**
+ * A Journey schema is a server schema that is saved in the Journey API
+ */
+
+type JourneySchemaWorkerData = {
```

Comment:
We tend to prefer using interfaces over type, great if you can change these to interfaces so we can keep it consistent.